src/tabular/grafter/tabular.clj

```diff
@@ -102,23 +104,23 @@
        (= ::not-found current-item-number)) []
 
        (= current-item-number index) (let [[repeated-item-numbers remaining-item-numbers]
-                                      (split-with #(= current-item-number %) item-numbers)
-                                      repeated-items (repeat (count repeated-item-numbers) current-item)]
-                                  (lazy-cat
-                                   repeated-items
-                                   (select-indexed item-data remaining-item-numbers)))
+                                           (split-with #(= current-item-number %) item-numbers)
+                                           repeated-items (repeat (count repeated-item-numbers) current-item)]
+                                       (lazy-cat
+                                         repeated-items
+                                         (select-indexed item-data remaining-item-numbers)))
 
        (< current-item-number index) (select-indexed
-                                     (drop-while (fn [[index item]]
-                                                   (not= index current-item-number))
-                                                 item-data)
-                                     rest-item-numbers)
+                                       (drop-while (fn [[index item]]
+                                                     (not= index current-item-number))
+                                                   item-data)
+                                       rest-item-numbers)
        (> current-item-number index) (select-indexed
-                                     (drop-while (fn [[index item]]
-                                                   (not= index current-item-number))
-                                                 item-data)
-                                     ;; leave item-numbers as is (i.e. stay on current item after fast forwarding the data)
-                                     item-numbers)))
+                                       (drop-while (fn [[index item]]
+                                                     (not= index current-item-number))
+                                                   item-data)
+                                       ;; leave item-numbers as is (i.e. stay on current item after fast forwarding the data)
+                                       item-numbers)))
 
```

Comment:
Bit of a indentation tidy up here ^^^. No functional change - just reads better.